docs/low-code-sql/development/gems/Transformations/aggregate.md

```diff
@@ -0,0 +1,76 @@
+
+![1](../img/Snow4.6.1_Aggregate.png)
+
+1. **Open** the HelloWorld_SQL [Project](https://app.prophecy.io/metadata).
+2. From the list of Models, select the **Orders** Model. A Model is a series of transformation steps (Gems) that describe how to create a single table or view. The Orders Model defines the steps to create the Orders table.
+3. Open the **Transformation** dropdown to see the available Transformation Gems. The Aggregate Gem has already been dragged to the canvas and configured in this HelloWorld_SQL example.
+
+Let's explore the Aggregate Gem by checking the input and output data samples, and then check how the Aggregate Gem is configured.
+
+4. Click the arrow to **Run upto the Aggregate Gem**.
+5. Preview a **data sample** before the Aggregate Gem.
+6. This is the `payments` data sample before the Aggregate Gem. There is one row per payment.
+7. Click to **Run upto the Join Gem**.
+8. Preview a **data sample** after the Aggregate Gem.
+9. This is the `order_payments` data sample after the Aggregate Gem. The individual rows of `payments` have been grouped according to `ORDER_ID`, and the amounts have been summed according to the payment type.
+10. Let's see how the Aggregate Gem is configured. Click to **Open**.
+
+![2](../img/Snow4.6.2_Aggregate.png)
+
+1. There is one **Input** Dataset, `payments`, and we can see the columns and datatypes below.
+2. Open the **Group By** tab.
+3. We can see the Gem is configured to group according to the `order_id` column. Just click any column name listed in **(1) Input** to add a column to the GroupBy expressions.
+4. Syntax **errors** are surfaced here as you're desiging your Gem (and Model) on the canvas. That's handy so you don't have to run a Job to discover a typo.
+5. The **Run** button is available here to test and view data samples. This way you can make sure your Aggregate Gem is configured as desired.
+
+Next we'll walk through the Aggregate tab, where we have a lot more bells and knobs to turn.
+
+![3](../img/Snow4.6.3_Aggregate.png)
+
+1. Click the **Aggregate** tab, where we'll define our new column names and data manipulation expressions.
+2. The list of **Expressions** describe how to manipulate a particular column from the input Dataset.
+3. The list of **Target Columns** is the list of column names to be manipulated by the Aggregate Gem and included in the Gem's output.
+4. Click **Output** to see the schema of the Dataset resulting from the Aggregate Gem.
+
+Let's go through several columns to see how the Aggregate Gem manipulates each column.
+
+5. The `order_id` column is getting passed through the Aggregate step without being changed. Recall this is the column that will be used to group the data. To add any column, just click the column name from the Input list, or start typing the column name and Prophecy Copilot will provide suggestions.
+6. Since the `order_id` column was (5) selected, then this column appears in the output Dataset. It has a number datatype.
```

Comment:
Just checking if the number call outs "(5)" and "(3)" are meant to be non-bolded.

Reply:
I originally did not bold because (5) and (3) are referring to previous steps. 

Let's try bolding and see if it's easier to read. This is a wall of text so far 😬